frameworks/Go/iris/benchmark_config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "framework": "iris",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "db_url": "/db",
+      "query_url": "/queries?queries=",
+      "update_url": "/update?queries=",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "Postgres",
+      "framework": "None",
+      "language": "Go",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "None",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "Iris",
+      "notes": "",
+      "versus": "go"
+    },
+    "postgresql": {
```

Comment:
Do you mean to have this here? You're already testing postgres in the default and there is no `iris-postgresql.dockerfile`

Reply:
No. I didn't. I had it there just to paste over the url links. I'll remove